setup.cfg

```diff
@@ -0,0 +1,5 @@
+[metadata]
```

Comment:
Would `.gitignore` this?

Reply:
sure, im not sure what of these setup files is typically ignored. If we remove someone else has to recreate? 

anyways i ignored it: [b6729ab](https://github.com/OCHA-DAP/ds-raster-stats/pull/7/commits/b6729aba9cbabbc4bedf7f82401b3fd2ff397ed9)

Reply:
Ok yes this makes sense! I didn't understand the connection with `pip install -e`